admin.js

```diff
@@ -109,20 +109,23 @@
     helpHook = function (moduleName, origionalHelp, config) {
         return function(commandPrefix, event) {
             if (getHasPermission(config, event.sender_id, event.sender_name, event.thread_id, moduleName)) {
-               return origionalHelp.call(this, commandPrefix);
+                return origionalHelp.call(exports.platform, commandPrefix);
             }
             return false;
         };
     };
 
 exports.load = function () {
     cfg = exports.config;
-    var loadedModules = this.modulesLoader.getLoadedModules('module');
+    var loadedModules = exports.platform.modulesLoader.getLoadedModules('module');
     for (var i = 0; i < loadedModules.length; i++) {
-        var match = loadedModules[i].match;
+        var match = loadedModules[i].match,
+            help = loadedModules[i].help;
+
         loadedModules[i].match = matchHook(loadedModules[i].__descriptor.name, match, cfg);
+        loadedModules[i].help = helpHook(loadedModules[i].__descriptor.name, help, cfg);
     }
-    this.modulesLoader.on('load', (event) => {
+    exports.platform.modulesLoader.on('load', (event) => {
```

Comment:
This listener is never removed, so admin and any module it touches will never be unloaded (and will require killing concierge to unload properly, a restart will not reload everything).